apps/docs/pages/guides/self-hosting.mdx

```diff
@@ -149,7 +149,7 @@ We realize that database migrations are difficult, and this is one of the proble
 
 While Supabase officially supports Docker, we have several other deployment strategies managed by the community:
 
-- [supabase-docker](../../guides/hosting/docker) (Official)
+- [supabase-docker](./docker) (Official)
```

Comment:
I think this makes the URL `https://supabase.com/docs/guides/docker` but it should be `https://supabase.com/docs/guides/self-hosting/docker` so I think it should just be this to avoid any ambiguity.
```suggestion
- [supabase-docker](/docs/guides/self-hosting/docker) (Official)
```